core/environment.go

```diff
@@ -79,6 +93,9 @@ func NewEnv(currentNs Symbol, stdin io.Reader, stdout io.Writer, stderr io.Write
 			:minor and/or :major, bugfix releases will increment :incremental.`, "1.0"))
 	res.args = res.CoreNamespace.Intern(MakeSymbol("*command-line-args*"))
 	res.SetEnvArgs(os.Args[1:])
+	res.classPath = res.CoreNamespace.Intern(MakeSymbol("*classpath*"))
+	res.classPath.Value = NIL
+	res.classPath.isPrivate = true
```

Comment:
Sadly, this is not enough: the var's meta needs to contain private keyword as well. See https://github.com/candid82/joker/blob/master/core/procs.go#L1550-L1558.